contracts/test/token/TestERC20.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.4;
+
+import "../../token/ERC20WithPermit.sol";
+
+contract TestERC20 is ERC20WithPermit {
+    string public constant NAME = "Test ERC20 Token";
```

Comment:
```suggestion
contract TestERC20WithPermit is ERC20WithPermit {
    string public constant NAME = "Test ERC20WithPermit Token";
```

Reply:
I assumed all our ERC20 tokens will have the permit functionality now and `TestERC20` name is just shorter for use in tests.

Reply:
But it may get confusing as for Open Zeppelin's `ERC721` we've got `TestERC721`. So `TestERC20` here may suggest that it's `OpenZeppelin's `ERC20` test token. For our test implementations, we usually use the same name as the contract with just `Test*` prefix, so maybe it would be worth doing the same here.